### DISCUSSION TITLE
Storage Template. Same named files.

### DISCUSSION THREAD

**Discussion by xstaford** [immich-app/immich#12445] [Feature Request]
### The issue with file name with storage template

After migrating files via storage template. {{y}}/{{yy}}{{MM}}{{dd}}_{{filetype}}-{{hh}}{{mm}}{{ss}} I have "same named" files with other register like date_time.JPG and date_time.jpg what is ok for Linux but not for a windows. So these files are not sync (back-up) with windows folder. 
The reason is because storage template repeats the extension as it is from original file. So as I mentioned above if the exension is in  capital, it is different file name for Immich. Thats why it doesnt add sequence number +x to the end.

Please make a way to change extension to small letters.

**Comment by mmomjian** [comment]
Thank you for this feature request. I think we have it tracked elsewhere but can’t find it.

**Comment by mmomjian** [comment]
Fixed in  #16667
